### PR TITLE
Remove credit

### LINKED ISSUE #395: Remove credit logic
TODO:

- [x] Remove `User.credit`, `User.max_credit`, `User.daily_credit`
- [x] Remove `'credit'` field of `User.traffic_history`
- [x] Remove JustGage
- [x] Remove credit-related stuff from `pycroft`, `hss`, `sample` models
- [ ] Remove translations involving the word `credit`

### PR DESCRIPTION
**Pull request by lukasjuhrich**
Fixes #395 
Left todo:
- [x] ~Make `warning` color thresholds in traffic table configurable~ Removed
- [x] ~Make sidebar link to traffic chart less ugly~ Declared “not ugly” by his lordship @Georgeto

### CODE REVIEW

File: sipa/blueprints/generic.py [hunk 1]
```diff
@@ -181,7 +181,9 @@ def logout():
 
 
 @bp_generic.app_template_filter('traffic_color')
-def traffic_color(amount, daily_credit):
+def traffic_color(amount):
+    # TODO think about other thresholds replacing `User.daily_credit`
+    daily_credit = 10 * 1024 * 1024
```
Comment (lukasjuhrich) [inline sipa/blueprints/generic.py]: @Georgeto thoughts about that?

**Comment by Georgeto** [comment]
>    Make sidebar link to traffic chart less ugly

I don't perceive the sidebar link to the traffic chart as ugly.

**Comment by Georgeto** [comment]
Had to rebase the branch to get the fix from ad71adcff875e18b1f794fb09306f79219c7a28f.

**Comment by lukasjuhrich** [comment]
> What could possibly go wrong?

**Comment by lukasjuhrich** [comment]
The PEP8-Failures should've been resolved by now, so the CI status should be green.

**Pull request merged**